apps/learner/src/routes/(app)/+page.svelte

```diff
@@ -33,33 +49,32 @@
   </div>
 
   <div class="flex flex-col gap-y-4">
-    <LearningUnit
-      to="/content/1"
-      tags={[{ variant: 'purple', content: 'Special Educational Needs' }]}
-      title="Navigating Special Educational Needs in Singapore: A Path to Inclusion"
-      showplayerpanel
-      onplay={handlePlay}
-    />
-
-    <LearningUnit
-      to="/content/1"
-      tags={[{ variant: 'purple', content: 'Special Educational Needs' }]}
-      title="Navigating Special Educational Needs in Singapore: A Path to Inclusion"
-      showplayerpanel
-      onplay={handlePlay}
-    />
+    {#each data.learningUnits as learningUnit (learningUnit.id)}
+      <LearningUnit
+        to={`/content/${learningUnit.id}`}
+        tags={[{ variant: 'purple', content: 'Special Educational Needs' }]}
+        title={learningUnit.title}
+        player={{
+          isactive: player.currentTrack?.id === learningUnit.id,
+          isplaying: player.isPlaying,
+          onplay: () => handleLearningUnitPlay(learningUnit),
+          onpause: handleLearningUnitPause,
+          onresume: handleLearningUnitResume,
+        }}
+      />
+    {/each}
   </div>
 </div>
 
 <div class="z-100 pointer-events-none fixed inset-x-0 bottom-0">
   <div class="mx-auto max-w-5xl px-4 py-3">
     <div class="flex justify-end gap-x-4">
-      {#if audioState.isFloatingPlayerVisible}
+      {#if player.currentTrack}
```

Comment:
Cool 🚀 , we can use the `stop()` to remove the current track and hide the floating player when the user takes a quiz.